scripts/job_bq_download.sh

```
@@ -0,0 +1,168 @@
+
+# ── Resume safety: stash chunks from a previous partial run ──────────────────
+# download_images.py restarts numbering at chunk_0001 and would overwrite them;
+# merge_sqfs_chunks.py searches recursively, so a subdir keeps them mergeable.
+if compgen -G "${TASK_DIR}/chunk_*.sqfs" > /dev/null; then
+    RESUME_DIR="${TASK_DIR}/resume_${SLURM_ARRAY_JOB_ID}"
+    mkdir -p "${RESUME_DIR}"
+    mv "${TASK_DIR}"/chunk_*.sqfs "${RESUME_DIR}/"
+    echo "Resume: moved $(ls ${RESUME_DIR} | wc -l) existing chunks to ${RESUME_DIR}"
+fi
+
+cd /project/6068129/melabbas/ami-ml
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)